### PR TITLE
UPDATES

### PR DESCRIPTION
**Pull request by yusuf-saif**
Heyyy i made some minimal changes,
added code of conduct docs and a contribution.md file
#20 #hactoberfest

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,29 @@
+### ToRoman
+[![Build Status](https://cloud.drone.io/api/badges/nytimes/library/status.svg)](https://cloud.drone.io/nytimes/library)
```
Comment (Zubs) [inline README.md:2]: What is this build status?

File: README.md [hunk 2]
```diff
@@ -12,15 +33,21 @@ A minimalist library for Roman numeral operations.
 - Get Roman numerals within a range 📡
 
 ## 📦 Installation
-
-It can be installed with npm.
+To get started with ToRoman,
+ you need NPM already installed or you can do
+```sh
```
Comment (Zubs) [inline README.md:38]: `npm install` to install `npm`? 💀

File: README.md [hunk 3]
```diff
@@ -1,8 +1,29 @@
...
+- [Features](#🚀Features)
+- [Installation](#📦Installation)
+- [Usage](#📥Usage)
+    - [Convert Integer to Roman Numerals](#convertintegertoromannumerals)
+     - [Convert  Roman Numerals to Integer](#convertromannumeralstointeger)
+- [Found this project useful](#Foundthisprojectuseful)
 # toroman
 
 A minimalist library for Roman numeral operations.
+A Minimal Library for Roman Numeral Operations.
```
Comment (Zubs) [inline README.md:17]: Duplicate sentence?

File: README.md [hunk 4]
```diff
@@ -1,8 +1,29 @@
...
+
+## CONTRIBUTION
+
+I know you find this project amazing and beautiful i know that feeling there but Please do take your time to 
+read our contributing.md docs first before you make your first contribution.
 
-## 🚀 Features
+oya abeg read our contributing.md docs first before you push rubbish.
```
Comment (Zubs) [inline README.md:24]: Why?

File: README.md [hunk 5]
```diff
@@ -12,15 +33,21 @@ A minimalist library for Roman numeral operations.
...
+  ```
+Next, type :
 
 ```sh
 npm i toroman
 ```
 
 ## 📥 Usage
 
+Using the ToRoman package is very simple and elaborate.
```
Comment (Zubs) [inline README.md:49]: Aren't simple and elaborate opposites?

File: README.md [hunk 6]
```diff
@@ -172,6 +199,9 @@ console.log(roman.range(22, 3, 5));
 // Returns [ 'III', 'VIII', 'XIII', 'XVIII' ]
 ```
 
+### DISCLAIMER
```
Comment (Zubs) [inline README.md:202]: What's this for?

File: contributing.md [hunk 7]
```diff
@@ -23,9 +23,9 @@ Pull requests are the best way to propose changes to the codebase (we use [Githu
 5. Make sure your code lints.
 6. Issue that pull request!
 
-## Any contributions you make will be under the MIT Software License
+## Any contributions you make will be under the  Software License
```
Comment (Zubs) [inline contributing.md:26]: MIT is the standard for Open Source. Or isn't it?

File: contributing.md [hunk 8]
```diff
@@ -23,9 +23,9 @@ Pull requests are the best way to propose changes to the codebase (we use [Githu
...
 
-In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project. Feel free to contact the maintainers if that's a concern.
+In short, when you submit code changes, your submissions are understood to be under the same Software License that covers the project. Feel free to contact the maintainers if that's a concern.
```
Comment (Zubs) [inline contributing.md:28]: Same as previous comment

File: contributing.md [hunk 9]
```diff
@@ -54,8 +54,12 @@ I'm again borrowing these from [Facebook's Guidelines](https://github.com/facebo
 
 ## License
 
-By contributing, you agree that your contributions will be licensed under its MIT License.
+By contributing, you agree that your contributions will be licensed under its Software License.
```
Comment (Zubs) [inline contributing.md:57]: Same as previous comment

**Review by Zubs — CHANGES_REQUESTED** [review]

**Comment by yusuf-saif** [comment]
Ok let me update them and make changes

File: README.md (hunk 1 above)
Comment (yusuf-saif) [reply README.md:2]: The build status is to show that it works

File: README.md (hunk 2 above)
Comment (yusuf-saif) [reply README.md:38]: Yes UX making it beginner friendly, incase someone wants to use and doesn't have NPM

File: README.md (hunk 3 above)
Comment (yusuf-saif) [reply README.md:17]: that's a mistake though

File: README.md (hunk 4 above)
Comment (yusuf-saif) [reply README.md:24]: just for fun let me remove that and leave the top one

File: README.md (hunk 5 above)
Comment (yusuf-saif) [reply README.md:49]: nope they aren't 
elaborate is well detailed
simple is easy and straightforward

**Pull request closed without merging**